test/mitmproxy/tools/web/test_app.py

```diff
@@ -480,6 +481,27 @@ def test_xsrf_hardening_login(self):
             ).body
         )
 
+    def login_test_helper(self, url, headers):
+        resp = self.fetch(url, headers=headers)
+        assert resp.code == 200
+        assert len(resp.headers.get_list("set-cookie")) > 0
+
+        set_cookies = {}
+        for cookie in resp.headers.get_list("set-cookie"):
+            set_cookies.update(httputil.parse_cookie(cookie.split(";", 1)[0]))
+
+        for key, value in httputil.parse_cookie(self.auth_cookie).items():
+            assert key in set_cookies
+            assert set_cookies[key] == value
```

Comment:
```suggestion
```

Status code is a good enough assertion here. And then we can inline this helper method. :)

Reply:
Fixed with 8dc7f34558e154e9489740f382852b0468926e24